src/components/BadgeButtonContainer.ts

```diff
@@ -150,6 +154,15 @@ export default class BadgeButtonContainer extends Component<BadgeButtonContainer
                     guids: [ mxObject.getGuid() ]
                 }
             });
+        } else if (onClickEvent === "callNanoflow" && nanoflow) {
+            window.mx.data.callNanoflow({
+                context,
+                error: (error: Error) => {
```

Comment:
Please follow the format on line 151

Reply:
Also apply similar feedback like for `boolean-slider` and `badge`

Reply:
Done